frontend/app/controllers/Info.scala

```diff
@@ -142,6 +145,7 @@ trait Info extends Controller {
         description = Some(CopyConfig.copyDescriptionSupporters)
       ),
       detailImageOrientated))
+      .withCookies(Cookie("gu_mem_ref_url", refUrl.getOrElse("")), Cookie("gu_refpvid", refPvid.getOrElse("")))
```

Comment:
Could you put `"gu_mem_ref_url"` and `"gu_refpvid" in string constants, as they're used elsewhere?

Separate thing- these two bits of code are equivalent, and I think the second one is a bit prettier, so I often use it in preference:

* `refUrl.getOrElse("")`
* `refUrl.mkString`